### ISSUE TITLE
Low contrast in new/edit poll question type

### ISSUE THREAD

**Issue by shailpatels** [Submitty/Submitty#7601]
**Describe the bug**
The poll question type descriptions are light gray on a white background and are getting flagged by WAVE and other checkers as having low contrast

**Expected behavior**
Contrast should meet WCAG criteria 

**To Reproduce**
1. Use a contrast checker like WAVE, or chrome/firefox's color picker tools

**Configuration**
 - Lightmode, any browser/os

**Screenshots**
<img width="1154" alt="image" src="https://user-images.githubusercontent.com/12129065/152072252-2068beb9-399c-47a6-9989-6c805b64cc0c.png">

Suggestion:
Use black as color for both options and labels, but increase the font-size of the options
<img width="1154" alt="image" src="https://user-images.githubusercontent.com/12129065/152073064-979a3c17-0d80-401d-9805-efb452078a3f.png">

**Additional context**
See 
https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast
https://submitty.org/developer/interface_design_style_guide/web_accessibility

for more details
